handlers_test.go

```diff
@@ -145,6 +153,31 @@ func TestGet(t *testing.T) {
 	require.NotEmpty(t, v.Headers)
 	require.NotEmpty(t, v.Origin)
 }
+func TestPost(t *testing.T) {
+	srv := testServer()
+	defer srv.Close()
+
+	data := `{[{"k1": "v1"}]}`
+
+	b := post(t, srv.URL+"/post?k1=v1&k1=v2&k3=v3", data)
+	v := struct {
+		Args    map[string]interface{} `json:"args"`
+		Headers map[string]string      `json:"headers"`
+		Origin  string                 `json:"origin"`
+		Data    string                 `json:"data"`
+		JSON    interface{}            `json:"json"`
+	}{}
+	require.Nil(t, json.Unmarshal(b, &v))
+	require.NotEmpty(t, v.Args, "args empty")
```

Comment:
this is redundant, you're checking v.Data below. you can just move the check below and replace this with that.

Reply:
True, I took this part directly  from the `TestGet`. Maybe this should be fixed to.